src/agni/client/receiver/ChatReceiver.java

```diff
@@ -2,4 +2,12 @@
 
 public class ChatReceiver extends MessageParser {
 
+    public ChatReceiver() {
+        super();
```

Comment:
The `super()` is unnecessary as it is implicitly called for a parent classes' default constructor.